.github/workflows/lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Lint
```

Comment:
we use pre-commit CI for our linting and it also allows us to fix pr's in place so this lint workflow is not needed